app/models/conversion_host.rb

```diff
@@ -75,12 +77,13 @@ def check_conversion_host_role
     tag_resource_as('disabled')
   end
 
-  def enable_conversion_host_role
+  def enable_conversion_host_role(v2v_vddk_package_url = nil)
     install_conversion_host_module
+    v2v_vddk_package_url ||= "http://#{resource.ext_management_system.hostname}/vddk/VMware-vix-disklib-stable.tar.gz"
```

Comment:
Minor nit, you're repeating "VMware-vix-disklib-stable.tar.gz" twice in this method. Maybe make some extra optional arguments, too?
```
  def enable_conversion_host_role(package_url = nil, package_name = nil, playbook = nil)
    install_conversion_host_module

    package_name ||= "VMware-vix-disklib-stable.tar.gz"
    package_url ||= "http://#{resource.ext_management_system.hostname}/vddk/#{package_name}"
    playbook ||= "/usr/share/ovirt-ansible-v2v-conversion-host/playbooks/conversion_host_enable.yml"

    extra_vars = {
      :v2v_vddk_package_name => package_name,
      :v2v_vddk_package_url  => package_url
    }

    ansible_playbook(playbook, extra_vars)
  ensure
    check_conversion_host_role
  end
```

Reply:
I am not trying to avoid touching the actual enable/disable implementations in this PR. IMHO, there are more other places need to be refactored.

Reply:
@jameswnl Ok, understood.